Grants docs

cmagan opened the pull request:
# Description

Update grants.md file adding overview, evaluation process and safes structure
Added a file application.md with some information about the application process

there are still links in the text that must get updated

# Changes

<!-- List of detailed changes (how the change is accomplished) -->

- [ ] ...
- [ ] ...

<!--
## Related Issues

Fixes #
-->

cmagan committed:
Update grants.md adding overview, evaluation process and safes structure

cmagan committed:
Create application

cmagan committed:
Rename application to application.md

cmagan committed:
Rename docs/governance/grants.md to docs/governance/grants/grants_overview.md

cmagan committed:
Rename docs/governance/application.md to docs/governance/grants/application.md

cmagan committed:
Create Grant Application Template

cmagan committed:
Rename Grant Application Template to application_template.md

cmagan committed:
Rename application.md to application_process.md

cmagan committed:
Update application_process.md

cmagan committed:
Update application_process.md

cmagan committed:
Update application_process.md

mfw78 committed:
chore: editorial review

mfw78 committed:
fix: links

mfw78 committed:
fixes: minor

mfw78 committed:
fix: navbar

mfw78 commented on docs/governance/grants/grants-program.md:
```diff
@@ -0,0 +1,93 @@
+
+As determined in [CIP-8](https://snapshot.org/#/cow.eth/proposal/0xdc641be107f139753cea051f1bacb8b74b915713a95306c3994f5e03e20d6bef), the CoW Grants Program Safe is controlled by two signers. Safe #3 is shared with the CoW DAO (Safe #1) and the Grants Committee (Safe #2). Safe #3 holds all the grant funds. You can check the governing process in the image below:
+
+(Todo: Create a mermaid chart of Safes including committee, grants-funds, volume-grants, cow-dao)
```
Please insert the mermaid chart before having this merged (or remove the todo)

mfw78 commented on docs/governance/grants/grants-program.md:
```diff
@@ -0,0 +1,93 @@
+
+(Todo: Create a mermaid chart of Safes including committee, grants-funds, volume-grants, cow-dao)
+
+The grants committee will have a distribution of 8 community members, making a total of 7 committee members. Safe #2 (from the schematic image above) [`gno:0xCA1F000D520c21C47E6c634DD31b92b91A6338bD`](https://app.safe.global/settings/setup?safe=gno:0xCA1F000D520c21C47E6c634DD31b92b91A6338bD) will function as the Grants committee Safe and has 8 signers with a signing threshold of 5 out of 8. The s ...
```
Confirm the number of committee / community members

mfw78 replied:
Are there 8 community members?

cmagan replied:
IMO this becomes increasingly irrelevant, we can just say the number of committee memebers

mfw78 replied:
If it's irrelevant, or seemingly irrelevant, we can remove it. It's just that "8 community members, making a total of 7 committee members" doesn't make sense?

mfw78 reviewed (commented):
Comments left in-line below. Also, for other reviewers, please have a once-over for grammar.

@cmagan Please sign the CLA per the comments above in the PR (just requires a simple "comment" to be made on your behalf)

cmagan commented:
I have read the CLA Document and I hereby sign the CLA

cmagan committed:
Update application-process.md

fix quorum number

cmagan committed:
Update grants-program.md

I'm not an expert on mermaid and this proved to be not as simple as I thought. LMK if this is reasonable

cmagan committed:
Update grants-program.md

cmagan committed:
Update grants-program.md

cmagan committed:
Update grants-program.md

fix typos

cmagan commented:
Hey @mfw78, I'm not sure I understand the remaining items to get this merged?

cmagan committed:
add timeline clause

cmagan committed:
add reference to the timeline clause

mfw78 commented on docs/governance/grants/application-process.md:
```diff
@@ -16,9 +16,23 @@ To apply for the CoW Grants program, use the [application template](application-
+
+To incorporate the adjustment you mentioned, here's a revised version of the clause that specifies the timeline is three months after the planned completion date as outlined in the grant proposal:
+
+---
```
Is this meant to be here?

mfw78 reviewed (commented):
Ideally this should also be reviewed by @gogonimago or @marshymarsh as well for copy

cmagan committed:
Remove unintended text

cmagan committed:
remove horizontal separator

cmagan committed:
Improve formatting and remove horizontal separators
